Dockerfile.debian

```diff
@@ -0,0 +1,102 @@
+# syntax = docker/dockerfile:1.2
+FROM docker.io/library/golang:1.19-bullseye AS builder
+
+RUN apt update
+RUN apt install -y build-essential git bash ca-certificates libstdc++6
+
+WORKDIR /app
+ADD go.mod go.mod
+ADD go.sum go.sum
+
+RUN go mod download
+ADD . .
+
+RUN --mount=type=cache,target=/root/.cache \
+    --mount=type=cache,target=/tmp/go-build \
+    --mount=type=cache,target=/go/pkg/mod \
+    make all
+
+
+FROM docker.io/library/golang:1.19-alpine3.16 AS tools-builder
```

Comment:
are you sure an alpine here?

Reply:
i think it would be best for tools to be build with same lib as where supposedly the db is created. 

used the golang alpine so that i could easily use go mod vendor